apps/dashboard/src/components/activity/activity-table.tsx

```diff
@@ -0,0 +1,172 @@
+import { useFetchActivities } from '../../hooks/use-fetch-activities';
+
+export interface ActivityTableProps {
+  selectedActivityId: string | null;
+  onActivitySelect: (activity: IActivity) => void;
+  filters?: IActivityFilters;
+}
+
+export function ActivityTable({ selectedActivityId, onActivitySelect, filters }: ActivityTableProps) {
+  const queryClient = useQueryClient();
+  const { currentEnvironment } = useEnvironment();
+  const hoverTimerRef = useRef<NodeJS.Timeout | null>(null);
+  const [searchParams] = useSearchParams();
+  const location = useLocation();
+  const navigate = useNavigate();
+  const { activities, isLoading, hasMore } = useFetchActivities({ filters });
+
+  const page = parseInt(searchParams.get('page') || '0');
+
+  const handlePageChange = (newPage: number) => {
+    const newParams = createSearchParams({
+      ...Object.fromEntries(searchParams),
+      page: newPage.toString(),
+    });
+    navigate(`${location.pathname}?${newParams}`);
+  };
+
+  const handleRowMouseEnter = (activity: IActivity) => {
+    hoverTimerRef.current = setTimeout(() => {
+      queryClient.prefetchQuery({
+        queryKey: [QueryKeys.fetchActivity, currentEnvironment?._id, activity._id],
+        queryFn: () => getNotification(activity._id, currentEnvironment!),
+      });
+    }, 1000);
+  };
+
+  const handleRowMouseLeave = () => {
+    if (hoverTimerRef.current) {
+      clearTimeout(hoverTimerRef.current);
+      hoverTimerRef.current = null;
+    }
+  };
+
+  useEffect(() => {
+    return () => {
+      if (hoverTimerRef.current) {
+        clearTimeout(hoverTimerRef.current);
+      }
+    };
+  }, []);
+
+  return (
+    <div className="flex min-h-full min-w-[800px] flex-1 flex-col">
+      <Table
+        isLoading={isLoading}
+        loadingRow={<SkeletonRow />}
+        containerClassname="border-x-0 border-b-0 border-t border-t-neutral-200 rounded-none shadow-none"
+      >
+        <TableHeader>
+          <TableRow>
+            <TableHead className="h-9 px-3 py-0">Event</TableHead>
+            <TableHead className="h-9 px-3 py-0">Status</TableHead>
+            <TableHead className="h-9 px-3 py-0">Steps</TableHead>
+            <TableHead className="h-9 px-3 py-0">Triggered Date</TableHead>
+          </TableRow>
+        </TableHeader>
+        <TableBody>
+          {activities.map((activity) => (
+            <TableRow
+              key={activity._id}
+              className={cn(
+                'relative cursor-pointer hover:bg-neutral-50',
+                selectedActivityId === activity._id &&
+                  'bg-neutral-50 after:absolute after:right-0 after:top-0 after:h-[calc(100%-1px)] after:w-[5px] after:bg-neutral-200'
+              )}
+              onClick={() => onActivitySelect(activity)}
+              onMouseEnter={() => handleRowMouseEnter(activity)}
+              onMouseLeave={handleRowMouseLeave}
+            >
+              <TableCell className="px-3">
```

Comment:
Missing the workflow icon for the code created workflows vs dashboard defined.
Btw please check the designs as we have a purple icon for the dashboard created ones.

Reply:
This was removed by Naveen, he forgot to update the designs

Reply:
But don't we need to have a way to differentiate?